kani-driver/src/cbmc_property_renderer.rs

```diff
@@ -240,6 +240,12 @@ pub fn format_result(properties: &Vec<Property>, show_checks: bool) -> String {
     let mut number_tests_undetermined = 0;
     let mut failed_tests: Vec<&Property> = vec![];
 
+    // cover checks
```

Comment:
Can you please add to the function documentation that we separate these two in the report?

Reply:
Done.